src/homepage/Components/ImpactData.js

```diff
@@ -1,20 +1,20 @@
 var ImpactData = [
-  {
+{
     id: 1,
+    name: "CodeMaps",
+    imgUrl: "image2.svg",
+    para1: "Learn everything the",
```

Comment:
para1: "Awesome roadmaps to learn",
 para2: "to code the right way!",

You have written the older content on the card. Change it